lang/csharp/src/apache/test/Avro.test.csproj

```diff
@@ -47,7 +47,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="$(MicrosoftNETTestSdkVersion)" />
-    <PackageReference Include="ReportGenerator" Version="5.0.4" />
+    <PackageReference Include="ReportGenerator" Version="$(ReportGenerator)" />
```

Comment:
Should this be a private asset?

Reply:
Is this really needed?

Reply:
After checking best practices for code coverage, IMO `ReportGenerator` should not be PackageReference'd

Reply:
https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-code-coverage?tabs=windows